### DISCUSSION TITLE
Help using replex without streaming going through replex

### DISCUSSION THREAD

**Discussion by jl94x4** [lostb1t/replex#96] [Q&A]
@eandrade92630 

I'm currently using nginx proxy manager to direct my plex ip:port to my domain at plex.domain.co.uk via ssl

**Comment by Zorro92** [comment]
Can you show what your nginx config currently looks like?

**Comment by jl94x4** [comment]
Sorry, I'm on mobile ATM formatting is horrendous.

**Comment by jl94x4** [comment]


**Reply by Zorro92** [reply]
So I'm kind of confused as to how this is working for you. Could you state what port you have Plex running on and what port you have replex running on? Usually the system also wouldn't let you use such a low port like port 90. I also don't see a proxy upstream for your catch all block "/'.

Usually it should look something like this:
server {
    listen 80;
    listen [::]:80;
    server_name example.com www.example.com;
    
    # All HTTP traffic is redirected to HTTPS
    location / {
        return 301 https://$host$request_uri;
    }
}

server {
    listen 443 ssl http2;
    listen [::]:443 ssl http2;
    server_name example.com www.example.com;

    ssl_certificate /etc/letsencrypt/live/example.com/fullchain.pem;
    ssl_certificate_key /etc/letsencrypt/live/example.com/privkey.pem;

    # Add more SSL configurations as needed (e.g., protocols, ciphers, etc.)

    # Proxy requests for /path1 to localhost:3001
    location /path1/ {
        proxy_pass http://localhost:3001;
        proxy_set_header Host $host;
        proxy_set_header X-Real-IP $remote_addr;
        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
    }

    # Proxy requests for /path2 to localhost:3002
    location /path2/ {
        proxy_pass http://localhost:3002;
        proxy_set_header Host $host;
        proxy_set_header X-Real-IP $remote_addr;
        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
    }

    # Proxy all other requests to localhost:3003
    location / {
        proxy_pass http://localhost:3003;
        proxy_set_header Host $host;
        proxy_set_header X-Real-IP $remote_addr;
        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
    }
}

**Comment by jl94x4** [comment]
@eandrade92630 I've tried formatting best i could in the post above!

**Comment by jl94x4** [comment]
My name on discord is thatja if that is any better as I can just send you the nginx config file instead of copy and pasting if it's easier that way :)

**Reply by Zorro92** [reply]
I've sent you a friend request

**Comment by jl94x4** [comment]
I'm running plex on 32400

Replex on port 90
